files/default/slurm/pcluster_slurm_config_generator.py

```diff
@@ -0,0 +1,160 @@
+import errno
+import json
+import logging
+from logging.handlers import RotatingFileHandler
+from os import makedirs, path
+from socket import gethostname
+
+import argparse
+
+from jinja2 import Environment, FileSystemLoader
+
+LOG_FILE_PATH = "/var/log/parallelcluster/pcluster_slurm_config_generator.log"
```

Comment:
do we want to have a log file specific for this script or simply log to stdout and let the caller have it logged? cfn-init already logs stdout by default if I'm not wrong.